# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ncftp/precompiled_headers.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -urN ncftp-3.1.9.orig/libncftp/pch.mk.gcc-gch ncftp-3.1.9/libncftp/pch.mk.gcc-gch
--- ncftp-3.1.9.orig/libncftp/pch.mk.gcc-gch	2003-05-17 05:05:25.000000000 +0200
+++ ncftp-3.1.9/libncftp/pch.mk.gcc-gch	2005-08-15 12:13:31.070899250 +0200
@@ -1,5 +1,5 @@
 syshdrs.h.gch: syshdrs.h ncftp.h ncftp_errno.h util.h ftp.h
-	$(CC) $(CFLAGS) $(CPPFLAGS) $(DEFS) syshdrs.h
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(DEFS) -c syshdrs.h
 
 PCH=syshdrs.h.gch
 SYSHDRS_DEP=syshdrs.h.gch
diff -urN ncftp-3.1.9.orig/sio/pch.mk.gcc-gch ncftp-3.1.9/sio/pch.mk.gcc-gch
--- ncftp-3.1.9.orig/sio/pch.mk.gcc-gch	2003-05-17 05:05:36.000000000 +0200
+++ ncftp-3.1.9/sio/pch.mk.gcc-gch	2005-08-15 12:12:53.856573500 +0200
@@ -1,5 +1,5 @@
 syshdrs.h.gch: syshdrs.h sio.h usio.h
-	$(CC) $(CFLAGS) $(CPPFLAGS) $(DEFS) syshdrs.h
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(DEFS) -c syshdrs.h
 
 PCH=syshdrs.h.gch
 SYSHDRS_DEP=syshdrs.h.gch
